src/aiortc/contrib/media.py

```diff
@@ -418,7 +420,9 @@ class MediaRecorder:
     :param options: Additional options to pass to FFmpeg.
     """
 
-    def __init__(self, file, format=None, options={}):
+    def __init__(self, file, format=None, options=None):
```

Comment:
This looks valid but unrelated?

Reply:
Oops, sorry you are right. It should be another PR.

Reply:
-> I moved these changes to #986

Reply:
.. and #986 is merged, thanks!